roles/sudoers/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# global file for sudoers
+
+# Role prefix is 'sudoers__'
+
+# Variables used as inputs in main.yml
+sudoers__env_admin_username:                     "{{ cdp_workload_user }}"
+sudoers__env_admin_password:                     "{{ globals.admin_password | mandatory }}"
```

Comment:
This is now a shared variable with https://github.com/cloudera-labs/cloudera.exe/blob/devel/roles/platform/defaults/main.yml#L66. Please pull this variable up into `cloudera.exe.common` and have both downstream roles map to this new upstream variable.

Reply:
Should be fixed in https://github.com/cloudera-labs/cloudera.exe/pull/6/commits/40d424a3a31f6f3bf9b6d2314bc811de5c5b2a8c